benchmark/src/main/scala/keycloak/scenario/KeycloakScenarioBuilder.scala

```diff
@@ -245,13 +245,47 @@ class KeycloakScenarioBuilder {
           status.is(200),
           // other elements like the access token not captured as we don't need them in the current scenarios and want to save memory
           jsonPath("$..id_token").find.saveAs("idToken"),
+          jsonPath("$..access_token").find.saveAs("accessToken"),
+          jsonPath("$..refresh_token").find.saveAs("refreshToken"),
+          jsonPath("$..expires_in").find.saveAs("expiresIn"),
         )
       )
+      // we must re-write these without the 'Secure' attribute, because otherwise it won't be submitted in future requests  (TODO -- or do we? this was a guess at the problem)
+      .exec(getCookieValue(CookieKey("AUTH_SESSION_ID").withDomain("192.168.49.2").withPath("/auth/realms/test-realm/").withSecure(true)))
```

Comment:
looks like we are assuming `test-realm` to be the only realm to fetch the cookie from ? and what is the reason behind setting specific cookies in the session ?

Reply:
it's in the associated Issue about the failing refresh operation, but that's fixed now and this was just temporary code that's been removed now

Reply:
great, thanks for taking care of that.